site/docs/tooltip.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+menu: docs
+title: Tooltip
+permalink: /docs/tooltip.html
+---
+
+Tooltip can provide details-on-demand about various marks on Vega-lite visualization. There are two distinct ways to create a tooltip
+
+* TOC
+{:toc}
+
+## Using Tooltip channel
+
+Vega-lite provided [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel to create tooltip using SVG [`title`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) tag. For example, this bar chart supports tooltip for field `b`. Hover over the bar will show a simple tooltip that displays individual value of field `b` of each bar.
+
+<div class="vl-example" data-name="bar_tooltip"></div>
+
+You can see that [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel provides a very limited functionality for tooltip since only one field can be displayed at a time. Moreover, no customization of the tooltip can be done using [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel.
```

Comment:
> You can see that [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel provides a very limited functionality

I know that you author the plugin so you're favoring the plugin more.  
However, i think we should adjust the tone here a bit.  "very limited" sounds very negative. :p
Let's say the benefit of this approach a bit --  something like: 

`tooltip` channel provides a quick and easy way to add a tooltip.  However, it is limited to displaying one field on the tooltip at a time.  

> Moreover, no customization of the tooltip can be done using [`tooltip`]({{site.baseurl}}/docs/encoding.html#mark-properties-channels) channel.

This is not precise.  Via Vega API, users can provide custom event handler.  Maybe remove this sentence and expand a bit on that.